coral-hive/src/main/java/com/linkedin/coral/transformers/CoralRelToSqlNodeConverter.java

```diff
@@ -0,0 +1,283 @@
+/**
+ * Copyright 2017-2022 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.transformers;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import org.apache.calcite.config.NullCollation;
+import org.apache.calcite.linq4j.tree.Expressions;
+import org.apache.calcite.rel.core.Correlate;
+import org.apache.calcite.rel.core.Join;
+import org.apache.calcite.rel.core.JoinRelType;
+import org.apache.calcite.rel.core.Project;
+import org.apache.calcite.rel.core.TableScan;
+import org.apache.calcite.rel.core.Uncollect;
+import org.apache.calcite.rel.logical.LogicalTableFunctionScan;
+import org.apache.calcite.rel.rel2sql.RelToSqlConverter;
+import org.apache.calcite.rel.type.RelDataType;
+import org.apache.calcite.rex.RexCall;
+import org.apache.calcite.rex.RexCorrelVariable;
+import org.apache.calcite.rex.RexFieldAccess;
+import org.apache.calcite.rex.RexNode;
+import org.apache.calcite.sql.JoinConditionType;
+import org.apache.calcite.sql.JoinType;
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlDialect;
+import org.apache.calcite.sql.SqlIdentifier;
+import org.apache.calcite.sql.SqlJoin;
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.SqlLateralOperator;
+import org.apache.calcite.sql.SqlLiteral;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.SqlUtil;
+import org.apache.calcite.sql.fun.SqlStdOperatorTable;
+import org.apache.calcite.sql.parser.SqlParserPos;
+import org.apache.calcite.util.Util;
+
+import com.linkedin.coral.com.google.common.collect.ImmutableList;
+import com.linkedin.coral.com.google.common.collect.ImmutableMap;
+import com.linkedin.coral.com.google.common.collect.Iterables;
+import com.linkedin.coral.hive.hive2rel.functions.HiveExplodeOperator;
+import com.linkedin.coral.hive.hive2rel.functions.HivePosExplodeOperator;
+
+import static org.apache.calcite.sql.parser.SqlParserPos.*;
+
+
+/**
+ * This class converts Coral RelNode to Coral SqlNode.
+ *
+ * This class currently handles only lateral view transformations
+ */
+public class CoralRelToSqlNodeConverter extends RelToSqlConverter {
+
+  public static final SqlDialect INSTANCE = returnInstance();
+
+  /**
+   * Creates a CoralRelToSqlNodeConverter.
+   */
+  public CoralRelToSqlNodeConverter() {
+    super(INSTANCE);
+  }
+
+  private static SqlDialect returnInstance() {
+    SqlDialect.Context context = SqlDialect.EMPTY_CONTEXT.withDatabaseProduct(SqlDialect.DatabaseProduct.HIVE)
+        .withNullCollation(NullCollation.HIGH);
+
+    return new SqlDialect(context);
+  }
+
+  /**
+   * This overridden implementation removes the catalog name from the table path.
```

Comment:
`removes the catalog part from the table namespace`

Reply:
fixed